span.go

```diff
@@ -91,6 +92,31 @@ func (s *spanImpl) SetTag(key string, value interface{}) opentracing.Span {
 	return s
 }
 
+func (s *spanImpl) LogKV(keyValues ...interface{}) {
```

Comment:
Yeah, this dates back to https://github.com/opentracing/opentracing.github.io/issues/96 .

The use cases haven't been well-centralized, but I feel good about having a timestamped KV  at the bottom.